Helpers/XmlHelper.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml;
+
+namespace ticksy
+{
+    public class XmlHelper
```

Comment:
If you want to Mock the XMLHelper, you need to use an Interface, and you don't really have an object-oriented design by putting a bunch of static methods in a class.